src/olympia/addons/forms.py

```diff
@@ -136,6 +137,20 @@ def get_tags(self, addon):
             return list(addon.tags.filter(restricted=False)
                         .values_list('tag_text', flat=True))
 
+    def clean(self):
```

Comment:
Hmm, I'm not sure if `AddonFormBase` is the right place for this. This get's inherited by a lot of forms, including `AddonFormMedia` where none of the checked values (slug, name, description) are changed.

Reply:
`name` and `summary are in Basic, but `description` is checked also and it's in Details

Reply:
How about extracting this `clean` method into a mixin that is configurable via a `akismet_checked_fields = ('name', 'summary')` property that can be inherited by forms actually needing it?

Reply:
https://github.com/mozilla/addons-server/pull/9353/commits/fda70c3c7dea08361420617dbe14d4e33513fae0